SDL/TrackCandidate.cu

```diff
@@ -328,14 +328,14 @@ __global__ void SDL::crossCleanpLS(struct SDL::modules& modulesInGPU, struct SDL
     }
 }
 
-__global__ void SDL::addpT3asTrackCandidatesInGPU(struct SDL::pixelTriplets& pixelTripletsInGPU, struct SDL::trackCandidates& trackCandidatesInGPU)
+__global__ void SDL::addpT3asTrackCandidatesInGPU(struct SDL::pixelTriplets& pixelTripletsInGPU, struct SDL::trackCandidates& trackCandidatesInGPU, struct SDL::segments& segmentsInGPU)
 {
     unsigned int nPixelTriplets = *pixelTripletsInGPU.nPixelTriplets;
 
     for(int pixelTripletIndex = blockIdx.x * blockDim.x + threadIdx.x; pixelTripletIndex < nPixelTriplets; pixelTripletIndex += blockDim.x*gridDim.x)
     {
-        if(pixelTripletsInGPU.isDup[pixelTripletIndex]) continue;
-
+//        if((!segmentsInGPU.isQuad[pixelTripletsInGPU.pixelSegmentIndices[pixelTripletIndex]]) || (pixelTripletsInGPU.isDup[pixelTripletIndex])) continue;//return;
```

Comment:
This is commented out and, if I understand correctly, this was the only place that `segmentsInGPU` were used in this kernel. If this is the case, should we drop this and also the extra argument in the kernel call?

Reply:
Yes, it's the only place that use the segmentsInGPU in this function. But I propose to leave it here because this commented out line can be used if we want to control the pixel is from quad, we can remove the comment and test here? probably that could be useful later?

Reply:
removed